routing/unified_edges.go

```diff
@@ -221,12 +221,8 @@ func calcCappedInboundFee(edge *unifiedEdge, amt lnwire.MilliSatoshi,
 
 	// Calculate the inbound fee charged for the amount that passes over the
 	// channel.
-	inboundFee := edge.inboundFees.CalcFee(amt)
-
 	// Take into account that the total node fee cannot be negative.
-	if inboundFee < -int64(nextOutFee) {
-		inboundFee = -int64(nextOutFee)
-	}
+	inboundFee := max(edge.inboundFees.CalcFee(amt), -int64(nextOutFee))
```

Comment:
Same here, the collapsed comments now look weird. I think we should keep the initial variable assignment and just replace the `if` statement with the `max` function.

Reply:
Thanks. Done